hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -482,3 +510,389 @@ def hpat_pandas_series_floordiv_number_impl(lhs, rhs):
         return hpat_pandas_series_floordiv_number_impl
 
     raise TypingError('{} The object must be a pandas.series or scalar. Given rhs: {}'.format(_func_name, rhs))
+
+
+@overload_method(SeriesType, 'pow')
+def hpat_pandas_series_pow(lhs, rhs, level=None, fill_value=None, axis=0):
+    """
+    Pandas Series method 'pow' implementation.
+        https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.pow.html#pandas.Series.pow
+
+    Algorithm: result = lhs.pow(other, level=None, fill_value=None, axis=0)
+
+    Where:
+               lhs: pandas.series
```

Comment:
Sorry but we changed documentation style. We have better formatting for docstrings now. Could you please apply new format style for all documentation?